modelerfour/test/scenarios/body-array/flattened.yaml

```diff
@@ -141,7 +141,7 @@ schemas: !<!Schemas>
       protocol: !<!Protocols> {}
     language: !<!Languages> 
       default:
-        name: FooEnum
+        name: Enum0
```

Comment:
Seems like this enum had a name before, any reason it'd be getting a generated name now?